src/common/Channel.cpp

```diff
@@ -191,6 +191,110 @@ void Channel::addOrReplaceTimeout(MessagePtr message)
     // WindowManager::instance().repaintVisibleChatWidgets(this);
 }
 
+void Channel::addOrReplaceSevenTvEventAddRemove(MessagePtr message)
```

Comment:
I'm not aware of the context of the PR but this function name is honestly pretty hard to understand

Reply:
We're following the `addOrReplaceTimeout` method name.